src/components/partner/partner.resolver.ts

```diff
@@ -93,6 +98,22 @@ export class PartnerResolver {
     );
   }
 
+  @ResolveField(() => SecuredFieldRegionList)
```

Comment:
Er...so we'd want to return `[SecuredFieldRegion]`?

Reply:
Here's what I'm trying now...but something is not right:

```TypeScript
 @ResolveField(() => [SecuredFieldRegion])
  async fieldRegions(
    @Parent() partner: Partner,
    @ListArg(FieldRegionListInput) input: FieldRegionListInput,
    @Loader(FieldRegionLoader) loader: LoaderOf<FieldRegionLoader>,
  ): Promise<SecuredFieldRegion[]> {
    return await mapSecuredValue(partner.fieldRegions, (ids) =>
      loader.loadMany(ids),
    );
  }
  ```
  
  I'm getting this type error on return:
  
  `Type 'Secured<(Error | FieldRegion)[]>' is missing the following properties from type 'SecuredFieldRegion[]': length, pop, push, concat, and 31 more.`

Reply:
The secured object is not a list.
```ts
@ResolveField(() => SecuredFieldRegion)

): Promise<SecuredFieldRegion> {
```
If the `value` of that type is a list, that's determined by that type. Which it should be.

The name should probably be plural or at least convey it's a list. Since we could have another 1-1 relation whose type would be `SecuredFieldRegion`.
I know we have another one that's a list. So idk here. Maybe just `SecuredFieldRegions`.

Reply:
That makes sense, `loader.loadMany(ids)` is returning a list.  So aren't we returning a list from the resolver?

Reply:
No the mapSecuredValue puts that in the `value` property. Though I'm just seeing that return null for value instead of empty list.
You could do 
```ts
    return {
      ...partner.fieldRegions,
      value: partner.fieldRegions.canRead ? await loader.loadMany(partner.fieldRegions.value) : [],
    };
```

Reply:
![Screenshot 2023-09-01 at 3 35 46 PM](https://github.com/SeedCompany/cord-api-v3/assets/1914742/d4f42f1e-8f52-4bc1-a3f6-ba4d68004f59)

🤨

Reply:
Try like this
https://github.com/SeedCompany/cord-api-v3/blob/master/src/components/scripture/dto/scripture-range.dto.ts#L135-L137